packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -244,6 +244,32 @@ describe('Combobox', () => {
             );
         });
 
```

Comment:
It's unlikely we'd implement it in a way that this would be a distinct test case from the other cases where at least one option is visible, but it doesn't hurt to have it.